src/ncp/ncp_base.cpp

```diff
@@ -1256,6 +1260,16 @@ otError NcpBase::CommandHandler_RESET(uint8_t aHeader)
     return error;
 }
 
+otError NcpBase::CommandHandler_REBOOT_BOOTLOADER(uint8_t aHeader)
+{
+    otError error = OT_ERROR_NONE;
+
+    otInstanceRebootBootloader(mInstance);
+    CommandHandler_RESET(aHeader);
+
+    return error;
```

Comment:
Suggestion: This is currently always returning `OT_ERROR_NONE`. How about returning the error returned from `CommandHandler_RESET()` instead?